inc/template-tags.php

```diff
@@ -457,9 +457,9 @@ function twentytwenty_body_classes( $classes ) {
 			$classes[] = 'has-full-width-content';
 		}
 
-		// Check for disabled search.
-		if ( get_theme_mod( 'twentytwenty_disable_header_search', false ) ) {
-			$classes[] = 'disable-search-modal';
+		// Check for enabled search.
+		if ( true === get_theme_mod( 'twentytwenty_enable_header_search' ) ) {
+			$classes[] = 'enable-search-modal';
```

Comment:
Does this class affect any CSS?

Reply:
@ joyously No, currently there is no CSS class `.disable-search-modal`. I believe @andersnoren added that body class to enable users to add conditional CSS, when needed, later on.

Reply:
OK, well, I just searched the files for 'search-modal' and the JS uses `search-modal`, which the CSS uses also. The header and the modal-search file both have the `search-modal` class, but also have data attributes 
`data-toggle-target=".search-modal"`
`data-toggle-body-class="showing-search-modal"`
`data-set-focus=".search-modal .search-field"`

Seems like the body class should match. (and 'showing' makes more sense than 'enable')